x/programs/rust/sdk_macros/src/lib.rs

```diff
@@ -64,11 +68,11 @@ pub fn public(_: TokenStream, item: TokenStream) -> TokenStream {
         }
     };
 
-    let param_idents = input_args
-        .iter()
-        .enumerate()
-        .skip(1)
-        .map(|(i, fn_arg)| match fn_arg {
+    let (input, context_type) = (input, context_type);
```

Comment:
nit: this is a noop

Reply:
Not quite, it's a move from mutable to immutable... but it's not obvious, so I'll make a change here